benefits/eligibility/views.py

```diff
@@ -131,9 +132,10 @@ def start(request):
     return TemplateResponse(request, "eligibility/start.html", ctx)
 
 
-@decorator_from_middleware(middleware.AgencySessionRequired)
-@decorator_from_middleware(middleware.RateLimit)
-@decorator_from_middleware(middleware.VerifierSessionRequired)
+@decorator_from_middleware(AgencySessionRequired)
+@decorator_from_middleware(LoginRequired)
```

Comment:
This is blocking Cypress from accessing `/eligibility/confirm`… Is that appropriate?